modules/afsql/afsql.h

```diff
@@ -28,7 +28,7 @@
 #include "mainloop-worker.h"
 #include "string-list.h"
 
-#include <dbi/dbi.h>
+#include <dbi.h>
 
```

Comment:
This is building upstream libdbi as well, installs it locally and works
with the above patch:
https://github.com/balabit/syslog-ng/pull/982
It uses libdbi-0.9.0 branch, though.

On Tue, Jun 21, 2016 at 10:52 PM, Balazs Scheidler <notifications@github.com
> wrote:

> In modules/afsql/afsql.h
> <https://github.com/balabit/syslog-ng/pull/1070#discussion_r67949461>:
>
> > @@ -28,7 +28,7 @@
> >  #include "mainloop-worker.h"
> >  #include "string-list.h"
> >
> > -#include <dbi/dbi.h>
> > +#include <dbi.h>
> >
>
> I am using upstream libdbi, so this might be a patch in Ubuntu then. I'll
> think it over how we can resolve my scenario then.
> … <#m_-9130651007225208046_>
> On Jun 21, 2016 11:11 AM, "Tamas Nagy" ***@***.***> wrote: In
> modules/afsql/afsql.h <#1070 (comment)
> <https://github.com/balabit/syslog-ng/pull/1070#discussion_r67833665>>: >
> @@ -28,7 +28,7 @@ > #include "mainloop-worker.h" > #include "string-list.h"
> > > -#include <dbi/dbi.h> > +#include <dbi.h> > This fix is valid on both
> Ubuntu Trusty and Xenial: `pkg-config --cflags dbi` …
> <#m_2827295804001033921_> -I/usr/include/dbi -I/usr/include/dbi/dbi Please
> open a new issue if you see a problem on a specific config and share
> details on the config on which this does not work as intended. On Tue, Jun
> 21, 2016 at 9:03 AM, Balazs Scheidler ***@***.***> wrote: In
> modules/afsql/afsql.h <#1070
> <https://github.com/balabit/syslog-ng/pull/1070> (comment) <#1070
> (comment)
> <https://github.com/balabit/syslog-ng/pull/1070#discussion_r67817095>>>:
> > @@ -28,7 +28,7 @@ > #include "mainloop-worker.h" > #include
> "string-list.h" > > -#include <dbi/dbi.h> > +#include <dbi.h> > This seems
> to have been the wrong fix. LIBDBI_CFLAGS is set to /usr/include in my case
> (e.g. it doesn't contain the dbi subdirectory), which comes from the .pc
> file installed by libdbi. Our configure script however encodes
> /usr/include/dbi in case the pc file is not found, which certainly needs
> this change. I think we should have changed configure.ac to match the pc
> file. I'll create a patch within my current branch to do that. — You are
> receiving this because you were assigned. Reply to this email directly,
> view it on GitHub <
> https://github.com/balabit/syslog-ng/pull/1070/files/ccc727cfaf6300f04bc63c42913172e00ee3384b..735c00b9be4e38a02a34883039f5e3aab26a88d0#r67817095>,
> or mute the thread <
> https://github.com/notifications/unsubscribe/AOobpMdxlZo4UoQmTXjtZWdPYYr3TPfVks5qN4y2gaJpZM4IoZ7j>
> . — You are receiving this because you modified the open/close state. Reply
> to this email directly, view it on GitHub <
> https://github.com/balabit/syslog-ng/pull/1070/files/ccc727cfaf6300f04bc63c42913172e00ee3384b..735c00b9be4e38a02a34883039f5e3aab26a88d0#r67833665>,
> or mute the thread <
> https://github.com/notifications/unsubscribe/AArldiewMccRSfjGGI4Zu5FZZSIzeaqXks5qN6rBgaJpZM4IoZ7j>
> .
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/balabit/syslog-ng/pull/1070/files/ccc727cfaf6300f04bc63c42913172e00ee3384b..735c00b9be4e38a02a34883039f5e3aab26a88d0#r67949461>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AOobpDQG-9g6ave51aksAi-CS_ySKescks5qOE8TgaJpZM4IoZ7j>
> .
>